src/compile/split.ts

```diff
@@ -68,6 +68,17 @@ export class Split<T extends object> {
       this.set(key, s[key], true);
     }
   }
+
+  /**
+   * Merge split object this split object. Properties from the other split
+   * overwrite properties from this split.
+   */
+  public copyAll(other: Split<T>) {
```

Comment:
I looked at `copyKeyFromObject` and `copyKeyFromObject` in the same class. What do you suggest? 

Yes, I want side effects because I don't want to have to return a copy everywhere.